server/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why this?

Reply:
Oh whoops. Dunno how that got in there. I've removed it.